src/types/tuple.rs

```diff
@@ -70,20 +70,19 @@ impl PyTuple {
     ///
     /// Panics if the index is out of range.
     pub fn get_item(&self, index: usize) -> &PyAny {
-        // TODO: reconsider whether we should panic
-        // It's quite inconsistent that this method takes `Python` when `len()` does not.
         assert!(index < self.len());
         unsafe {
-            self.py()
-                .from_borrowed_ptr(ffi::PyTuple_GET_ITEM(self.as_ptr(), index as Py_ssize_t))
+            // PyTuple_GET_ITEM return borrowed ptr; must make owned for safety (see #890).
+            let ptr = ffi::PyTuple_GET_ITEM(self.as_ptr(), index as Py_ssize_t);
```

Comment:
Do we need this trick even for tuple? :thinking:

Reply:
It's less critical for tuple I think, because it's immutable from Python - but it's possible to mutate a tuple in C / Rust!

I felt for extra peace of mind I would make this owned too. I can undo this one if you prefer.

Reply:
> I can undo this one if you prefer.

I don't think it's the problem of whether I prefer or do not prefer it.
Since we don't have an API for deleting an object(= `del_item` returns an error), I don't see a clear advantage for using `owned` here.

Reply:
Ok. 👍